plugins/generator-datapack-1.17.x/datapack-1.17.x/jsontriggers/block_placed.json.ftl

```diff
@@ -2,7 +2,9 @@
   "trigger": "minecraft:placed_block",
   "conditions": {
     "item": {
-      "item": "${input$block}"
+        "items": [

```

Comment:
Items are nested in item here, is this ok?

Reply:
Yes, this is how it works. Strange, but working.